module/VuFind/tests/integration-tests/src/VuFindTest/Mink/AccountMenuTest.php

```diff
@@ -0,0 +1,349 @@
+        $menu = $page->findAll('css', '#login-dropdown');
+        $this->assertEquals(0, count($menu));
+        $stati = $page->findAll('css', '.account-menu .fines-status.hidden');
+        $this->assertEquals(0, count($stati));
+
+        // Nothing on
+        $this->changeConfigs(
+            [
+                'config' => [
+                    'Authentication' => [
+                        'enableAjax' => false,
+                        'enableDropdown' => false
+                    ]
+                ]
+            ]
+        );
+        $session->reload();
+        $this->snooze();
+        $page = $session->getPage();
+        $menu = $page->findAll('css', '#login-dropdown');
+        $this->assertEquals(0, count($menu));
+        $stati = $page->findAll('css', '.account-menu .fines-status.hidden');
+        $this->assertEquals(1, count($stati));
+
+        // Menu off, dropdown on
+        $this->changeConfigs(
+            [
+                'config' => [
+                    'Authentication' => [
+                        'enableAjax' => false,
+                        'enableDropdown' => true
+                    ]
+                ]
+            ]
+        );
+        $session->reload();
+        $this->snooze();
+        $page = $session->getPage();
+        $menu = $page->findAll('css', '#login-dropdown');
+        $this->assertEquals(1, count($menu));
+        $stati = $page->findAll('css', '.account-menu .fines-status.hidden');
+        $this->assertEquals(2, count($stati)); // one in menu, one in dropdown
+
+        // Reset all on
+        $this->changeConfigs(
+            [
+                'config' => [
+                    'Authentication' => [
+                        'enableAjax' => true,
+                        'enableDropdown' => true
+                    ]
+                ]
+            ]
+        );
+        $session->reload();
+        $this->snooze();
+        $page = $session->getPage();
+        $menu = $page->findAll('css', '#login-dropdown');
+        $this->assertEquals(1, count($menu));
+        $stati = $page->findAll('css', '.account-menu .fines-status.hidden');
+        $this->assertEquals(0, count($stati));
+    }
+
+    public function testCacheClearing()
+    {
+        $session = $this->getMinkSession();
+        $session->visit($this->getVuFindUrl());
+        $this->snooze();
+        // Seed some fines
+        $this->setJSStorage(['fines' => ['value' => 30.5, 'display' => '$30.50']]);
+        // Clear different cache
+        $session->evaluateScript('VuFind.account.clearCache("holds");');
+        $cookies = $this->getJSStorage();
+        $this->assertEquals(true, $cookies['fines'] != null);
+        // Clear correct cache
+        $session->evaluateScript('VuFind.account.clearCache("fines");');
+        $cookies = $this->getJSStorage();
+        $this->assertEquals(true, $cookies['fines'] == null);
+
+        /* TODO: Not working in tests, but working in browser
```

Comment:
Is this something you need help troubleshooting?

Reply:
Sure! Each individual clearCache seems to work (ie. `clearCache("fines")`) but the universal one (no argument passed) wasn't working on my server during the test.